doc/version_1/047_treesForVulMgmt_4.md

```diff
@@ -71,7 +71,7 @@ Once the decision points are selected and the prioritization labels agreed upon,
 Making the decision process accessible has a lot of benefits.
 Unfortunately, it also makes it a bit too easy to overcomplicate the decision.
 
-The SSVC version 1 ~applier~ deployer tree had 225 rows when we wrote it out in long text form.
+The SSVC version 1 deployer tree had 225 rows when we wrote it out in long text form.
```

Comment:
I had this in strikeout (~tilde~) on purpose to make it clear that it was called something else in v1, so if a reader goes to look at the v1 doc they know it was called applier then, though we have changed it to deployer now. So I think this should stay. Sorry, I didn't catch this earlier.